lib/zookeeper.js

```diff
@@ -694,6 +753,11 @@ class ZooKeeper extends EventEmitter {
         return NativeZk.ZOOKEEPER_READ;
     }
 
+    /** @deprecated @returns {string} "/zookeeper/config" */
```

Comment:
I think this should have been added to the `constants.js` file instead, do direct users there instead. Long term, I would like to remove all code marked as deprecated in the source code,.

I will add it in a separate PR.